test/event-repository.test.js

```diff
@@ -0,0 +1,20 @@
+/* eslint-disable no-undef, no-unused-expressions */
+const expect = require('chai').expect;
+const { writeMessage } = require('../repositories/event-repository');
+
+describe('Event Repo writeMeaage', () => {
+  it('should exist', () => {
+    expect(writeMessage).to.exist;
+  });
+
+  it('should be a function', () => {
+    expect(writeMessage).is.a('function');
+  });
+
+  it('should return an array', () => {
+    writeMessage('U1111111', 'Message text', '', 'C1111111')
```

Comment:
@Dave-Gallup is it the case that this test will need to hit the DB on the other side of the `knex` connection? If so I think you will need to take care in this file to set the `NODE_ENV` to the appropriate setting so that you can test against your test db.